sappho/tilemap.py

```diff
@@ -67,28 +67,26 @@ def parse_rules(path_to_rules_file):
                 that are set
 
         """
-        
-        with open(path_to_rules_file) as f:
-            rules = f.readlines()
-
         tile_rules = {}
 
-        for rule in rules:
-            tile_ids_affected, flags = rule.split('=')
-            tile_ids_affected = [id_ for id_ in tile_ids_affected.split(',')]
-            flags = [flag.strip() for flag in flags.split(',')]
+        with open(path_to_rules_file) as f:
```

Comment:
It's faster nesting, and it makes more sense from my point of view.

Considering `file.readlines()` is equivalent to the following:

```python
for line in f:
    yield line
```

It's much faster to optimize out that full iteration when we're iterating over lines in the first place.